ui/pages/home/home.component.js

```diff
@@ -345,8 +346,49 @@ export default class Home extends PureComponent {
       setWaitForConfirmDeepLinkDialog(false);
     });
     ///: END:ONLY_INCLUDE_IF
+
+    this.triggerNewPrivacyPolicyToast();
   }
 
+  triggerNewPrivacyPolicyToast() {
+    const newPrivacyPolicyDate = new Date(2024, 5, 6);
```

Comment:
Check out `ui/helpers/constants/survey.ts` for how I handle and store the date information